pkg/vm/createEvm.go

```diff
@@ -142,6 +147,22 @@ func createEvmGenesis(app *application.Avalanche, subnetName string, subnetEVMVe
 	return prettyJSON.Bytes(), sc, nil
 }
 
+func ensureAdminsHaveBalance(admins []common.Address, alloc core.GenesisAlloc) error {
+	if len(admins) < 1 {
+		return nil
+	}
+
+	for _, admin := range admins {
+		// we can break at the first admin who has a non-zero balance
+		if bal, ok := alloc[admin]; ok &&
+			bal.Balance != nil &&
+			bal.Balance.Uint64() > uint64(0) {
+			return nil
+		}
+	}
+	return errors.New("could not find any admin in the TxAllowListConfig which is funded")
```

Comment:
Let's make this error message a little more helpful. Maybe `None of the address in the transaction allow list precompile have any tokens allocated to them. Currently, no address can transact on the network. Airdrop some funds to one of the allow list addresses to continue`.